extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/AuthenticationProviderService.java

```diff
@@ -205,38 +214,80 @@ private LDAPConnection bindAs(Credentials credentials)
      *     denied.
      */
     public AuthenticatedUser authenticateUser(Credentials credentials)
-            throws GuacamoleException {
-
-        // Attempt bind
-        LDAPConnection ldapConnection;
-        try {
-            ldapConnection = bindAs(credentials);
-        }
-        catch (GuacamoleException e) {
-            logger.error("Cannot bind with LDAP server: {}", e.getMessage());
-            logger.debug("Error binding with LDAP server.", e);
-            ldapConnection = null;
-        }
-
-        // If bind fails, permission to login is denied
-        if (ldapConnection == null)
-            throw new GuacamoleInvalidCredentialsException("Permission denied.", CredentialsInfo.USERNAME_PASSWORD);
-
-        try {
-
-            // Return AuthenticatedUser if bind succeeds
-            AuthenticatedUser authenticatedUser = authenticatedUserProvider.get();
-            authenticatedUser.init(credentials);
-            return authenticatedUser;
-
-        }
-
-        // Always disconnect
-        finally {
-            ldapService.disconnect(ldapConnection);
-        }
-
+    		throws GuacamoleException {
+
+    	// Attempt bind
+    	LDAPConnection ldapConnection;
```

Comment:
This should be indented with spaces, not tabs. It should be 4 spaces per indentation level.a